### ISSUE TITLE
Can't enter dungeon from overworld

### ISSUE THREAD

**Issue by bunnylin** [bradhannah/Ultima5Redux#251]
Trying to E-nter a dungeon from the overworld results in a non-fatal Ultima5ReduxException: [dungeon name]: 0 was not found!

It should of course expect a Word of Power and then let you in.

**Comment by cambragol** [comment]
This is what I am seeing too.

Also, do we have a list of what should be working in the release? Did I miss that? Because it would help with testing.

**Comment by bunnylin** [comment]
Also, shrines can't be entered, but of course implementing those is a bigger thing since shrines have special functionality.

I'm guessing this release is mostly just for having a look around, giving us plebs a chance to be awed. :D The issue list looks like it'll keep development busy for a while even without new issues added.

**Comment by bradhannah** [comment]
Yep! I actually thought it was a softlock - I know it was during my late testing.  I have a ticket for words of power - I also do not currently observe if the word had already been uttered or if the path is blocked by a rock.

**Comment by bradhannah** [comment]
moving out of milestone because you basically have to implement dungeons first!
